docs/keyboard.md

```diff
@@ -224,3 +224,51 @@ static scheduleLayoutAnimation(event)
 ```
 
 Useful for syncing TextInput (or other keyboard accessory view) size of position changes with keyboard movements.
+
+## Type Definitions
+
+### KeyboardMetrics
+`KeyboardMetrics` object is part of [`Keyboard`](text#addListener) callback and contains the measurement data of `Keyboard`.
```

Comment:
We also use this in a static function `Keyboard.metrics()` (that I forgot to add to documentation when I added the API...). Instead of linking to the list of APIs using this, could we instead link to this from that API?

I.e. add a reference to `KeyboardMetrics` in the `addListener` section, but replace the KeyboardMetrics description with a description of what the type is?

```suggestion
`KeyboardMetrics` represents the size and position of an on-screen keyboard.
```

`on-screen` and `virtual` usually mean the same thing, except for VR which can have a soft keyboard outside of screen-space. So the metrics are *technically* the intersection of the keyboard and the screen. But I don't know how much detail we need about that case.